packages/portal/src/components/browse/BrowseSections.vue

```diff
@@ -50,6 +50,13 @@
         :text="section.text"
         :url="section.url"
       />
+      <CallToActionBanner
+        v-else-if="contentType(section, 'PrimaryCallToAction') && section.image"
```

Comment:
why is the image required here?

Reply:
for the primary CTA component to be used otherwsie. 

Styling for that has been updated.